tests/test_modeling.py

```
@@ -399,159 +354,134 @@ def basic_pairs_setup(self):
             tail_idx = pair_idx[0, 0, 1].item()
-            
+
             if head_idx >= 0 and tail_idx >= 0:
                 assert torch.allclose(head_rep[0, 0], span_rep[0, head_idx])
                 assert torch.allclose(tail_rep[0, 0], span_rep[0, tail_idx])
-    
+
     def test_respects_threshold(self, basic_pairs_setup):
         """Should exclude pairs below threshold."""
         # Test with high threshold
         pair_idx_high, pair_mask_high, _, _ = build_entity_pairs(
```

Ruff (0.15.1)

[warning] Unpacked variable `pair_idx_high` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)